geomstats/geometry/hyperboloid.py

```diff
@@ -392,7 +383,7 @@ def log(self, point, base_point):
             Tangent vector at the base point equal to the Riemannian logarithm
             of point at the base point.
         """
-        angle = self.dist(base_point, point) / self.scale
+        angle = self.dist(base_point, point)
```

Comment:
Removing here does not affect the result? That is, are you getting the expected result with `ScalarProductMetric`?

Reply:
Yes - it works. I was able to put the removed tests back in but use the `ScalarProductMetric` functionality. So I'll keep doing that with as many removed tests as possible.